.changeset/many-bags-press.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': minor
+---
+
+Fix issue causing incorrect sorting behavior for autogenerated sidebars
```

Comment:
And while I’m here I guess we can also update this to Sarah’s preferred verb tense:

```suggestion
Fixes incorrect sorting behavior for some autogenerated sidebars
```